### PR TITLE
use the nextcloud certificate bundle for s3

### PR DESCRIPTION
**Pull request by icewind1991**
Allows using self signed certificates with s3 storage by adding them to the nextcloud cert bundle.

### CODE REVIEW

**Review by CarlSchwan — APPROVED** [review]

File: lib/private/Files/ObjectStore/S3ConnectionTrait.php [hunk 1]
```diff
@@ -120,6 +121,9 @@ public function getConnection() {
 			)
 		);
 
+		/** @var ICertificateManager $certManager */
+		$certManager = \OC::$server->get(ICertificateManager::class);
```
Comment (nickvergessen) [inline lib/private/Files/ObjectStore/S3ConnectionTrait.php:125]: Thi aparently also depends on the filesystem:
https://github.com/nextcloud/server/pull/31360

could we come into trouble? or is this not for primary storages?

Comment (icewind1991) [reply lib/private/Files/ObjectStore/S3ConnectionTrait.php]: disabled it from primary storage

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**